File: Java/CountWords.java
//  To count words in a sentence

import java.util.Scanner;

public class CountWords {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a sentence: ");
        String str=sc.nextLine();
        int count=0;
        boolean flag=false;
        for(int i=0;i<str.length();i++){
            if(str.charAt(i)!=' ' && !flag){
                count++;
                flag=true;
            }
            else if(str.charAt(i)==' '){
                flag=false;
            }
        }
        if(count==0){
                System.out.println("No words");
            }else{
                 System.out.println("Total words are: "+count);
            }
    }
}

File: python/CountWords.py
# To count words in a sentence
str=input("Enter a sentence: ")
count=0
flag=False
for i in str:
    if i!=" " and not flag:
        count+=1
        flag=True
    elif i==" ":
        flag=False
if count==0:
    print("No Words")
else:
    print("Total words are: ",count)

    